official-package/aws-organizations/.aws-sam/build/template.yaml

```diff
@@ -0,0 +1,230 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: CrowdStrike Falcon SSM Distributor for AWS Organizations
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+    - Label:
+        default: Deployment Scope
+      Parameters:
+      - ProvisionOU
+      - Regions
+    - Label:
+        default: Falcon API Credentials
+      Parameters:
+      - FalconClientID
+      - FalconSecret
+      - FalconCloud
+    - Label:
+        default: Credential Storage Options
+      Parameters:
+      - SecretStorageMethod
+      - SecretsManagerSecretName
+      - FalconClientIdParameter
+      - FalconSecretParameter
+      - FalconCloudParameter
+    - Label:
+        default: Falcon Distributor Configuration
+      Parameters:
+      - AutomationAssumeRole
+      - ProvisionOU
+      - Action
+      - ScheduleExpression
+      - MaxErrors
+      - MaxConcurrency
```

Comment:
Can't remember, but was there a reason we didn't let them set install time values like `WindowsInstallerParams`, `LinuxInstallerParams`, etc